cmd/run/run_test.go

```diff
@@ -573,6 +575,9 @@ func TestBuildServerWithOIDCAuthentication(t *testing.T) {
 		Issuer:   localOIDCServerURL,
 	}
 
+	// release the oidc server port after obtaining cfg ports
+	oidcServerPortReleaser()
```

Comment:
i don't understand why this is here. at this point in the code, the test hasn't run. should we run this code as a last step?

Reply:
We need to release the port before we call RunServer.  Otherwise, we try to use a port that is in use (i.e., net.Listen in line 299 of cmd/run/run.go and cause RunServer to fail.

Reply:
Something seems off to me.. the OpenFGA server and the mock OIDC server should run in different ports, no?

Reply:
The issue is that server will fail due to OIDC port already in use.